omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -65,45 +130,174 @@ configuration redirect the output of the following command into a file:
 To configure an HTTPS server follow
 `the nginx documentation <http://nginx.org/en/docs/http/configuring_https_servers.html>`_.
 
-Start the Gunicorn worker processes running one thread listening on 127.0.0.1:4080:
+Running OMERO.web (Unix/Linux)
+------------------------------
+
+Start the Gunicorn worker processes listening by default on 127.0.0.1:4080:
 
 ::
 
     $ bin/omero web start
-    ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
+    ... static files copied to '/home/omero/OMERO.server/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
-Additional settings can be configured from command line arguments:
+The Gunicorn workers are managed **separately** from other OMERO.server
+processes. You can check their status or stop them using the
+following commands:
 
-.. program:: omero web start
+::
+
+    $ bin/omero web status
+    OMERO.web status... [RUNNING] (PID 59217)
+    $ bin/omero web stop
+    Stopping OMERO.web... [OK]
+    Django WSGI workers (PID 59217) killed.
 
-.. option:: --workers WORKERS
+.. _download_limitation:
 
-    The number of worker processes for handling requests.
+Download limitations
+^^^^^^^^^^^^^^^^^^^^
 
-.. option:: --worker-connections WORKER_CONNECTIONS
+In order to offer users the ability to download data from OMERO.web you have
+to deploy using :ref:`gunicorn_async_configuration`.
+OMERO.web is able to handle multiple clients on a single worker
+thread switching context as necessary while streaming binary data from
+OMERO.server. Depending on the traffic and scale of the repository you should
+configure connections and speed limits on your server to avoid blocking
+resources. We recommend you run benchmark and performance tests.
+It is also possible to apply :ref:`download_restrictions` and
+offer alternative access to binary data.
 
-    The maximum number of simultaneous clients.
+.. note::
+    Handling streaming request/responses requires proxy buffering
+    to be turned off. For more details refer to
+    `Gunicorn deployment <http://docs.gunicorn.org/en/stable/deploy.html>`_.
 
 .. note::
-    Additional wsgi arguments can be set using property
-    :property:`omero.web.wsgi_args`
+    :property:`omero.web.application_server.max_requests` should be set to 0
 
-    ::
 
-        $ bin/omero config set omero.web.wsgi_args -- "--forwarded-allow-ips=127.0.0.1"
+.. _nginx_wsgi_benchmark:
 
-    For more details check
-    `Gunicorn Documentation <http://docs.gunicorn.org/en/latest/settings.html>`_.
+Benchmark
+---------
 
-The Gunicorn workers are managed **separately** from other OMERO.server
-processes. You can check their status or stop them using the
-following commands:
+We run example banchmarks on rendering thumbnail and 512x512 pixels plane
```

Comment:
Singular and plural mixed here.  Maybe "benchmarks to render thumbnails and 512×512 pixel planes for 100 concurrent users, making 5000 requests in total::"